### PR TITLE
Adjust for breaking changes in hwtracer.

### PR DESCRIPTION
**Pull request by ptersilie**
This fixes yk to work with the changes in https://github.com/softdevteam/hwtracer/pull/81.

### CODE REVIEW

File: yktrace/Cargo.toml [hunk 1]
```diff
@@ -11,7 +11,7 @@ elf = "0.0"
 fallible-iterator = "0.2"
 ykpack = { path = "../ykpack" }
 lazy_static = "1.3"
-hwtracer = { git = "https://github.com/softdevteam/hwtracer" }
+hwtracer = { git = "https://github.com/ptersilie/hwtracer", branch="refactor" }
```
Comment (vext01) [inline yktrace/Cargo.toml:14]: did you mean to commit this hunk?

Comment (ptersilie) [reply yktrace/Cargo.toml:14]: Whoops. No!

Comment (ptersilie) [reply yktrace/Cargo.toml]: Fixed.

**Comment by ptersilie** [comment]
Can I squash?

**Comment by vext01** [comment]
oops, yes please do

**Comment by ptersilie** [comment]
Squashed.

**Comment by vext01** [comment]
```
error[E0599]: no method named `first_instr` found for struct `hwtracer::Block` in the current scope
  --> yktrace/src/hwt/mapper.rs:36:44
   |
36 |                     let start_addr = block.first_instr() - self.phdr_offset;
   |                                            ^^^^^^^^^^^ method not found in `hwtracer::Block`
error[E0599]: no method named `last_instr` found for struct `hwtracer::Block` in the current scope
  --> yktrace/src/hwt/mapper.rs:37:42
   |
37 |                     let end_addr = block.last_instr() - self.phdr_offset;
   |                                          ^^^^^^^^^^ method not found in `hwtracer::Block
```

**Comment by ptersilie** [comment]
Huh, now that the other PR is merged this should work. Let's make sure we waited long enough.

bors try

**Comment by vext01** [comment]
This is weird. I was able to reproduce this locally and deleting `Cargo.lock` fixed it.

The CI clones afresh and `Cargo.lock` is not in git, so I don't see why we see that error...

**Comment by vext01** [comment]
bors try

**Comment by vext01** [comment]
tried killing `~/.cargo` in all workers.

bors try

**Comment by vext01** [comment]
OK, well that's baffling.

Let's keep an eye on this...

bors r+

**Pull request merged**